community/community.css

```diff
@@ -0,0 +1,134 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+header{
+    display: flex;
+    font-size: 1.3rem;
+}
+.prev{
+    color: #000000;
+    font-size: 2rem;
+    margin-left:20px ; margin-top: -5px;
+}
+.nav-header{
+    border-bottom: 1px solid #b3b3b3;
+}
+.title{
+    display: flex;
+    padding-top: 8px;
+    margin-bottom: 6px;
+    border-bottom: 1px solid #b3b3b3;
+}
+.heading h1{
+    text-align: center;
+    font-size: 2rem;
+    color: #000000;
+    margin: 0 auto;
+}
+.filter {
+    width: 25%;
+    padding: 13px 14px 2px 2px;
+    border-right: 1px solid #b3b3b352;
+    text-align: center;
+    color: #3b3b5ab7;
+}
+.filter i{
+    color: #3b3b5ab7;
+    padding-right: 3px;
+}
+.scroll {
+    width: 75%;
+}
+.item {
+    padding: 10px 14px 2px 10px;
+    font-size: 1.3rem;
+}
+.nav-header a {
+    text-decoration: none;
+    color: #343435;
+}
+.nav-header a:hover{
+    color: #FF8527;
+
+}
+.filter,.item,.scroll {
+    display: inline-block;
+}
+.scroll {
+    white-space: nowrap;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+    -ms-overflow-style: -ms-autohiding-scrollbar;
+}
+.scroll::-webkit-scrollbar {
+    display: none;
+}
+.categories{
+    display:inline-block;
+    width:min-content;
+    height:fit-content;
+    border-radius: 10px;
+    padding: 0;
+    margin: 15px 0px 0px 12px;
+
+}
+.categories .first {
+    position: absolute;
+    left: 160px;
+    top: 250px;
+}
+.categories .second {
+    position: absolute;
+    left: 360px;
+    top: 250px;
+}
+.categories .third {
+    position: absolute;
+    left: 160px;
+    top: 566px;
+}
+.categories .fourth {
+    position: absolute;
+    left: 360px;
+    top: 566px;
+}
+.categories .fifth {
+    position: absolute;
+    left: 160px;
+    top: 875px;
+}
+.categories .sixth {
+    position: absolute;
+    left: 360px;
+    top: 875px;
+}
+.categories h1{
+    font-size: 1.5rem;
+    padding: 4px 0px 4px 3px;
+    width: 180px;
+}
+.categories img{
+    padding-left: 3px;
+    padding-right: 3px;
+    width: 180px;
+    height: 180px;
+    position: relative;
+}
+.people img{
+    width: 50px;
+    height: 50px;
+}
+.people{
+    display: inline-flexbox;
+    padding-bottom: 8px;
+}
+.people h3 , span{
```

Comment:
It would be nice if you use the design fonts, which are Recoleta and Cera pro. Otherwise, You rocked it Marwa ^.^

Reply:
@import url("http://fonts.cdnfonts.com/css/recoleta");
@import url("http://fonts.cdnfonts.com/css/cera-pro");

Add them in the CSS file and change each font family that matches the design text fonts.